docs/flink-integration-guide.md

```diff
@@ -78,7 +78,7 @@ limitations under the License.
     val carbonProperties = new Properties
     // Set the carbon properties here, such as date format, store location, etc.
      
-    // Create carbon bulk writer factory. Two writer types are supported: 'Local' and 'S3'.
+    // Create carbon bulk writer factory. Three writer types are supported: 'Local', Hdfs' and 'S3'.
```

Comment:
Everywhere if we use file factory API and support HDFS conf input, only one writer is enough right ? do we need 3 writers ?
Because in carbon table or SDK we don't create multiple type of writers to handle this kind of scenario.

Reply:
yes. i also thought about the same. But since, already they have implemented writers for LOCAL and S3 type, i have implemented for HDFS. But i can see, there are some differences only for S3 writer, some extra configurations are needed and they are not creating directory while writing stage directories in S3. you can check  CarbonS3Writer.commit.

Reply:
Chaned code to use  CarbonLocalWriter itself to handle Local and Hdfs FileSystems. Please review